src/main/java/com/icthh/xm/uaa/service/TenantRoleService.java

```diff
@@ -514,15 +514,16 @@ private void enrichExistingPermissions(
 
     private Optional<String> getConfigContent(String configPath) {
         String tenant = TenantContextUtils.getRequiredTenantKeyValue(tenantContextHolder.getContext());
-        String config = null;
+        String config;
         try {
             config = tenantConfigRepository.getConfigFullPath(tenant, API + configPath);
             if (StringUtils.isBlank(config) || EMPTY_YAML.equals(config)) {
                 config = null;
             }
 
         } catch (HttpClientErrorException e) {
-            log.warn("Error while getting '{}'", configPath, e);
+            log.warn("Error while getting '" + configPath + "'", e);
```

Comment:
I'm not a snob, but why not use templating ({})? It should log exception as well as I know

Reply:
agree, it is still not clear for me